packages/shared/lib/tests/api.test.ts

```diff
@@ -98,7 +98,7 @@ describe('File: api.test.ts', () => {
     describe('Function: createStardustAccount', () => {
         it('should call createStardustAccount', async () => {
             spy = jest.spyOn(profileManagerMock, 'createAccount')
-            const payload = { alias: 'alias' }
+            const payload = { alias: 'alias', coinType: 4219 }
```

Comment:
Although this is corresponding to the Shimmer token type, is there an enum we can use here rather than a magic constant?

Reply:
Ahh this is just temporary until we flesh out the coinType side of the app logic and creating accounts. So will be reworked in a later pr.